spec/lib/listen/adapter/darwin_spec.rb

```diff
@@ -69,130 +70,4 @@
       it { should eq 1234 }
     end
   end
-
-  describe 'multiple dirs' do
```

Comment:
Specs are no longer relevant and there is already high level acceptance test that runs for OSX on CI.

Reply:
Oh, I see. Thanks.